components/PrivacyPolicy/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react'
+
+import Cookies from 'js-cookie'
+import Router from 'next/router'
+import PropTypes from 'prop-types'
+
+import { api, setHeader } from '../../utils/axios'
+import styles from '../GlobalComponents/Modal/styles'
+
+const PrivacyPolicy = ({ credentials }) => {
+  const [modal, setModal] = useState(false)
+  const [reject, setReject] = useState(false)
+
+  useEffect(() => {
+    if (credentials.email && !credentials.dataPolicyAccepted) {
+      setHeader(credentials)
+      setModal(true)
+    }
+  }, [])
+
+  const handleReject = async () => {
+    await api.delete(`/user/${credentials._id}`)
+    setModal(false)
+    Cookies.remove('ctech_credentials')
+    Router.push('/')
+  }
+
+  const handleAccepted = async () => {
+    await api.patch(`/user/dataPolicyAccepted/${credentials._id}`)
+    setModal(false)
+  }
+
+  const getContent = () => {
+    if (reject) {
+      return content.reject
+    } else {
+      return content.default
+    }
```

Comment:
@marianaviana um ternário resumiria bastante aqui
```suggestion
      return reject ? content.reject : content.default
```